okhttp/src/main/java/io/grpc/okhttp/AsyncFrameWriter.java

```diff
@@ -35,10 +35,10 @@
   private Socket socket;
   // Although writes are thread-safe, we serialize them to prevent consuming many Threads that are
   // just waiting on each other.
-  private final SerializingExecutor executor;
+  private final Executor executor;
```

Comment:
Does the above comment still apply on line 36?

If we didn't use a serializing executor here and instead used a threadpool, couldn't enqueued items appear to run out of order due to thread scheduling?